### PR TITLE
Insures users have tree to visulaize their project, fixes #2

### LINKED ISSUE #2: insure users have "tree"
Include a sentence making sure users have "tree" so they can visualize the project like Python Model Structure -> Basic Structure

### PR DESCRIPTION
**Pull request by dnoci001**
A comment about visualizing your project structure with tree has been added

### CODE REVIEW

File: _episodes/02-structure.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ Let us layout our first module like the following:
 └── .gitignore                      <- Stock helper file telling git what file name patterns to ignore when adding 
 ```
 
+To visualize your project like above you will use "tree"
+(linux) if you do not have tree you can get using $sudo apt-get install tree
```
Comment (dgasmith) [inline _episodes/02-structure.md]: Can we change "$sudo apt-get install tree" to "\`sudo apt-get install tree\`". So that we get highlight like `this`.

**Comment by bennybp** [comment]
Very good. You should you backticks rather than apostrophes, though (see comment from @dgasmith)

**Comment by dnoci001** [comment]
backticks have been added

**Comment by bennybp** [comment]
Looks much better. Thanks

**Pull request merged**